renderdoc/os/posix/posix_process.cpp

```diff
@@ -315,13 +348,16 @@ static pid_t RunProcess(const char *app, const char *workingDir, const char *cmd
   pid_t childPid = fork();
   if(childPid == 0)
   {
-    chdir(workDir.c_str());
-
-    // in child process, so we can change environment
-    environ = envp;
-    execvp(appName.c_str(), argv);
-
-    RDCERR("Failed to execute %s: %s", appName.c_str(), strerror(errno));
+    const string appPath(GetAbsoluteAppPathFromName(appName));
+    if(!appPath.empty())
+    {
+      chdir(workDir.c_str());
+      execve(appPath.c_str(), argv, envp);
+      RDCERR("Failed to execute '%s' %s", appName.c_str(), strerror(errno));
+      RDCERR("Full Path:'%s'", appPath.c_str());
+    }
+    else
+      RDCERR("Failed to execute '%s' Executable not found", appName.c_str());
```

Comment:
I don't mind brace-less ifs or elses if it's just a one-line short statement, but I prefer it to be consistent so if there are braces for the if case here then add braces for any else cases.